src/redux/constants.js

```diff
@@ -0,0 +1,3 @@
+export const NEWS_REQUESTED = 'NEWS_REQUESTED';
+export const NEWS_GOT = 'NEWS_GOT';
+export const NEWS_FAILED = 'NEWS_FAILED';
```

Comment:
Bad naming, it is not clear whether you receive news or send it. REQUESTED SUCESS REJECTED